server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -378,7 +378,7 @@ trait EmailQueryMethodContract {
         AppendCommand.builder().build(message2))
       .getMessageId
 
-    val requestDateMessage3 = Date.from(ZonedDateTime.now().minusDays(1).toInstant)
+    val requestDateMessage3 = Date.from(ZonedDateTime.now().minusDays(3).toInstant)
```

Comment:
idem you inversed the order of element by sent at. But did not change the assertion

Reply:
Can we rewrite this test with the same message ordering than `listMailsShouldBeSortedByDescendingOrderOfSentAt`

IE:

 - message1 sent at now().minusDays(3)
 - message2 sent at now().minusDays(2)
 - message3 sent at now().minusDays(1)

And have the assertion `.isEqualTo(s"""["${messageId1.serialize()}", "${messageId2.serialize()}", "${messageId3.serialize()}"]""")` ?

Reply:
**That's not done**